website/content/en/docs/overview/cheat-sheet.md

```diff
@@ -0,0 +1,86 @@
+linkTitle: "Cheat Sheet"
+weight: 3
+description: >
+    Operator-SDK Cheat Sheet commands and operations
+---
+
+Following a cheat sheet with the options and helpers for projects which are built with SDK and are respecting its proposed layout.
+
+## Base commands and options 
+
+| Helper   | Description  |
+|-------|-----------|
+| `$ operator-sdk init`          | To initialize a operator project in the current directory |
+| `$ operator-sdk init --plugins=<plugin-key>`          | To initialize a operator project in the current directory using an specific plugin. For you are able to check the available plugins you can run `oparator-sdk --help`. E.g (`operator-sdk init --plugins=helm`)|
+| `$ operator-sdk create api [flags]`          | Let you create your own APIs with its [GKV][gkvs] by [Extendig the Kubernetes API with CustomResourceDefinitions][extend-k8s-api], or let you use external or core-types as well as generate their respectively [controllers][controllers-k8s-doc]|
+| `$ operator-sdk create webhook [flags]`          | To scaffold [Webhooks][webhooks-k8s-doc] for the APIs declared onn the project. Note that only Go-based project supports this option currently.|
+| `$ operator-sdk init --plugins=<plugin-name>`|           
+| `$ make docker-build IMG=<some-registry>/<project-name>:<tag>`          | Build the operator image      |
+| `$ make docker-build docker-push IMG=<some-registry>/<project-name>:<tag>`      | Build and push the operator image for your registry  |
+| `$ make install`         | Install the CRDs into the cluster |
+| `$ make uninstall`         | Uninstall the CRDs into the cluster |
+| `$ make run`         | Run your controller locally and outside of the cluster. Note that, this will run in the foreground, so switch to a new terminal if you want to leave it running. |
+| `$ make deploy`         | Deploy your project on the cluster |
+| `$ make undeploy`         | Undeploy your project on the cluster |
+
+## To work with bundles and help you out to integrate your project with OLM
+
+For further information check [Operator SDK Integration with Operator Lifecycle Manager][olm-integration].
+
+| Helper   | Description  |
+|-------|-----------|
+| `$ make bundle`          | Create/update the [bundle][bundle] based on the project manifests in the `bundle/` directory. More info [Create a bundle][creating-a-bundle]      |
+| `$ operator-sdk bundle validate ./bundle`          | To validate your [bundle][bundle] spec definition.      |
+| `$ operator-sdk bundle validate ./bundle --select-optional suite=operatorframework` | Validate your bundle against [OperatorHub.io][operatorhub-io] criteria. For further information use the flag.`--help` |
+| `$ operator-sdk olm install` | To install OLM on your cluster for development purposes |
+| `$ operator-sdk olm uninstall` | To uninstall OLM from your cluster |
+| `$ make bundle-build IMG=<some-registry>/<project-name-bundle>:<tag>` | To build your bundle operator image |
```

Comment:
Add commands that aren't obvious from `make help` or reading the Makefile
```suggestion
```

Reply:
I am not sure if is obvious for any user. We know that we can combine them. However, would any person or a new user know that as well?